afdko/Tools/SharedData/FDKScripts/BezTools.py

```diff
@@ -1,3 +1,16 @@
+from __future__ import print_function, absolute_import
+import sys
+
+if sys.version_info > (3,):
+	long = int
+	longType = long
```

Comment:
why do you need both `longType` (lowercase initial) and `LongType` (uppercase), both alias of `int` on py3?

Reply:
oh fixed 14446c2a79eaa5e1a5bf843c904e200f9515e167